src/pages/studies/index.js

```diff
@@ -12,6 +11,13 @@ export default function StudyIndex(props) {
 
   return (
     <main className="study-index">
+      {/* Page Metadata */}
```

Comment:
case study index: added metadata, linked the first two cards to `/studies/light-two` and `/studies/sample` respectively